scss/reprivata-theme/custom.scss

```diff
@@ -264,9 +268,30 @@ $accordion-body-padding-bottom: $spacer + 8;
 
 //Form Control
 $input-border-color: $black;
-$input-font-size: $font-size-sm;
+$input-font-size: $caption-font-size;
 $input-padding-y: $spacer / 2;
 $input-padding-x: $spacer / 2;
+$input-line-height: $input-btn-line-height;
+$input-focus-border-color: $blue;
+$input-focus-border: map-get($border-widths, 2) solid;
```

Comment:
Changing border widths on focus results in the inner dimensions of the control changing. This is generally discouraged. 

Instead, applying a pseudo-border using `box-shadow` should be used instead. Here's an example of what I'm talking about: https://codepen.io/aarondancer/pen/dyNjXLZ

- Outline: Notice how using `offset` produces a clean inset border on the control that doesn't affect the control's dimensions. The outline acts almost like it's overlaying the control. 
- Outline - Offset: To make the outline appear outset instead of inset, we can apply an `outline-offset`. The problem with this is it usually produces a blurred outline instead of solid. 
- Border:  Notice how when you focus on the control, there's some jank. This is because the control's box is resizing to accommodate the border. So, if you have a row of controls (like a search box + button), you'll face misalignment issues.
- Border - Fixed Height: Now, if we apply a fixed height to the control, the jank related to box resizing doesn't apply. however, there's still jank caused by the control itself resizing to accommodate the box dimensions specified. This can lead to text not rendering properly. 
- Box Shadow: Produces a pseudo-border using `box-shadow`. It's a little soft on the edges in some browsers, but it produces a decent result that's outset.
- Box Shadow - Inset: Produces the same pseudo-border, but inset in the control instead of outset. 

If you're using a Chromium-based browser, the outline examples probably looks fine. However, if you switch to Firefox, outlines look totally different. This is why most projects use box-shadow instead. Box shadows produce the most consistent behavior across browsers without the drawbacks of using a border.